packages/samples/stencil/.eslintrc.js

```diff
@@ -5,7 +5,6 @@ const gtsPath = resolve(gtsPkgJsonPath, '..');
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
```

Comment:
Fixes a eslint bug that was present in multiple packages

`ESLint couldn't determine the plugin "@typescript-eslint" uniquely.
`